packages/office-ui-fabric-react/src/components/DatePicker/examples/DatePicker.Basic.Example.tsx

```diff
@@ -74,16 +74,12 @@ export class DatePickerBasicExample extends React.Component<{}, IDatePickerBasic
     };
   }
 
-  private _timeChangeCallback(date: Date) {
-    console.log(date);
-  }
-
   public render() {
     const { firstDayOfWeek } = this.state;
 
     return (
       <div>
-        <DatePicker displayDatePickerFormat={ DatePickerFormat.dateOnly } setSelectedDateTime={ this._timeChangeCallback } firstDayOfWeek={ firstDayOfWeek } onAfterMenuDismiss={ () => console.log('onAfterMenuDismiss called') } allowTextInput strings={ DayPickerStrings } showGoToToday isMonthPickerVisible placeholder='Select a date...' />
```

Comment:
Can we add onAfterMenuDismiss={ () => console.log('onAfterMenuDismiss called') }  back?

Reply:
We can as long as it's not using the lambda abbreviated form, I can just add a short function to replace it?

Reply:
They should be able to take lambda though, does it give you compiling errors?

Reply:
Yep